apps/full-site-editing/full-site-editing-plugin/editor-plans-grid/src/plans-modal/styles.scss

```diff
@@ -0,0 +1,33 @@
+@import '~@automattic/typography/sass/fonts';
```

Comment:
Just FYI, if https://github.com/Automattic/wp-calypso/pull/43718 gets merged first, this line need rebasing.